subcommittees/mentoring/ITProposal.md

```diff
@@ -0,0 +1,99 @@
+# Suggestions to Instructor Training
+
+**This is a rough draft.**
+
+## Terminology
+
+-   Checkout session: 1h meeting where learners at Instructor Training delivery
+    their last task.
+
+    The meetings are lead by Instructor Trainers and can have at maximum 6
+    learners.
+-   Mini-lesson: 1h lesson for at least 3 people delivering part of one lesson
+    from Software or Data Carpentry curriculum.
+-   Session: 3h lesson for at least 10 people delivering one full lesson
+    from Software or Data Carpentry curriculum.
+-   Workshop: 12h lesson for at least 20 people delivering the full curriculum
+    of Software or Data Carpentry.
+-   Instructor in Training: can taught and organize a mini-lesson, session or workshop under
+    the supervision of a Software or Data Carpentry instructor.
+-   Instructor: can lead, taught and organize a mini-lesson, session or workshop.
+
+## Badges
+
+Anyone interested in join Software and Data Carpentry team of instructors
+**must** complete the Instructor Training in any format offered by Software and
+Data Carpentry.
+
+To complete the Instructor Training is required to accomplished one final exercise
+describe below.
+
+After the final exercise, Software and Data Carpentry give the "Instructor in
+Training" badge for those that complete the final exercise.
+
+"Instructor in Training" has one year to taught one mini-lesson,
+one session or one workshop under the supervision of a Software or Data
+Carpentry Instructor.
+
+The Software or Data Carpentry Instructor **can** recommend the "Instructor in
+Training" to receive the Instructor badge from the Software or Data
+Carpentry Executive Diretor.
+
+## Final Exercises
+
+1.  Watch a 5min lesson video of a Software or Data Carpentry instructor
+    about one of the lessons on the currículum.
+
+2.  Sign for one checkout session.
+
+3.  Join the checkout session.
+
+4.  Delivery a 5min lesson about one of the lessons on the curriculum.
```

Comment:
Make it clear that the trainee gets to decide which lesson, but we get to decide which segment of that lesson they have to teach, so that we know they've mastered the whole thing.